### ISSUE TITLE
dev script not working in Windows 10

### ISSUE THREAD

**Issue by omnibusinc** [diegohaz/arc#62]
trying to run the 'dev' task in windows was failing for me.
I needed to update the task to the following:
"dev": "concurrently --kill-others \"nodemon --watch src\" \"node webpack/webpack-dev-server\"",

**Comment by diegohaz** [comment]
It doesn't seem to me as valid json. Did you mean this?
```json
"dev": "concurrently --kill-others \"nodemon --watch src\" \"node webpack/webpack-dev-server\"",
```

**Comment by omnibusinc** [comment]
Sorry, yes.  Github stripped out the backslashes in my comment.

**Comment by diegohaz** [comment]
Fixed in 	bf3ab80
